industrial_ci/src/tests/source_tests.sh

```diff
@@ -126,8 +126,11 @@ fi
 
 ici_time_start rosdep_install
 
+if [ "${ROSDEP_INSTALL_OPTS// }" == "" ]; then
```

Comment:
This `if` clause only defines a variable. So can it be moved to [industrial_ci/src/env.sh](https://github.com/ros-industrial/industrial_ci/blob/83f6c9ba12fdeb7333a73a72d221f681d263640e/industrial_ci/src/env.sh)?
Cc @ipa-mdl